### PR TITLE
81 disallow future revocations

### PR DESCRIPTION
**Pull request by gargravarr**

### CODE REVIEW

File: src/main/java/org/zalando/planb/revocation/config/properties/RevocationProperties.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ *   {@code issued_before} is equal to the current
+ * </ul>
+ *
+ * @author  <a href="mailto:rodrigo.reis@zalando.de">Rodrigo Reis</a>
+ */
+@ConfigurationProperties(prefix = "revocation")
+@Data
+public class RevocationProperties {
+
+    private Integer timestampThresold = 5;
```
Comment (hjacobs) [inline src/main/java/org/zalando/planb/revocation/config/properties/RevocationProperties.java]: small typo here

File: src/main/java/org/zalando/planb/revocation/api/impl/RevocationResourceImpl.java [hunk 2]
```diff
@@ -94,18 +117,40 @@ public RevocationList get(@RequestParam(required = true) final int from) {
...
+            case CLAIM:
+                timestamp = ((RevokedClaimsData) revocation.getData()).getIssuedBefore();
+                break;
+            case GLOBAL:
+                timestamp = ((RevokedGlobal) revocation.getData()).getIssuedBefore();
+                break;
         }
+
+        // Checks for future timestamps
+        if (timestamp > UnixTimestamp.now() + revocationProperties.getTimestampThresold()) {
```
Comment (hjacobs) [inline src/main/java/org/zalando/planb/revocation/api/impl/RevocationResourceImpl.java]: timestamp should default to "NOW" if it's null

**Comment by vroldanbet** [comment]
:+1:

**Comment by vroldanbet** [comment]
:+1:

File: src/main/java/org/zalando/planb/revocation/api/impl/RevocationResourceImpl.java (hunk 2 above)
Comment (gargravarr) [inline src/main/java/org/zalando/planb/revocation/api/impl/RevocationResourceImpl.java]: It does, when the object is created, see https://github.com/zalando/planb-revocation/blob/master/src/main/java/org/zalando/planb/revocation/domain/RevokedClaimsData.java#L27

File: src/main/java/org/zalando/planb/revocation/config/properties/RevocationProperties.java (hunk 1 above)
Comment (gargravarr) [inline src/main/java/org/zalando/planb/revocation/config/properties/RevocationProperties.java]: fixed

**Pull request merged**